managed/services/preconditions.go

```diff
@@ -45,21 +46,41 @@ func CheckMongoDBBackupPreconditions(q *reform.Querier, mode models.BackupMode,
 		}
 
 		for _, task := range tasks {
-			if task.ID != scheduleID {
-				return status.Errorf(codes.FailedPrecondition, "A PITR backup for cluster '%s' can be enabled only if there no other scheduled backups for this cluster.", clusterName)
+			if task.ID == scheduleID {
+				// Needed for scheduled PITR backup update.
+				continue
```

Comment:
Could you describe when we need such behaviour?

Reply:
Well, it's in the comment, I will add more desciptive comment. It's needed in when you want to update existing PITR backup, this function should pass.

Reply:
Updated, but not sure if it become better actually.